lib/brains/views.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from lib.brains.client import get_client
+
+
+@api_view(['POST'])
```

Comment:
yup, but thanks for reminding me: https://github.com/mozilla/solitude/issues/372